CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.3.1 (09/10/2018)
+- Fix long-running regex match
+
 ## 0.2.1 (14/12/2017)
```

Comment:
Oh, what happened to the `0.3.0` entry in the `Change Log`?

Reply:
i assume it hasn't been there

Reply:
Yah I missed that 👎